src/app/index.ts

```diff
@@ -1,2 +1,29 @@
-export * as Wallets from "./wallets"
-export * as Users from "./users"
+import * as AccountsMod from "./accounts"
+import * as AdminMod from "./admin"
+import * as LightningMod from "./lightning"
+import * as PricesMod from "./prices"
+import * as UsersMod from "./users"
+import * as WalletsMod from "./wallets"
+import { wrapAsyncToRunInSpan, wrapToRunInSpan } from "@services/tracing"
+
+const allFunctions = {
+  Accounts: { ...AccountsMod },
+  Admin: { ...AdminMod },
+  Lightning: { ...LightningMod },
+  Prices: { ...PricesMod },
+  Users: { ...UsersMod },
+  Wallets: { ...WalletsMod },
+}
+const syncFunctions = ["PaymentStatusChecker"]
+
+for (const subModule in allFunctions) {
+  for (const fn in allFunctions[subModule]) {
+    const wrapper = syncFunctions.includes(fn) ? wrapToRunInSpan : wrapAsyncToRunInSpan
```

Comment:
only `PaymentStatusChecker` is sync? why not making this function async in that case so we don't have to go through this condition?

Reply:
I didnt want to change the implementations but code is more cleaner. Updated here https://github.com/GaloyMoney/galoy/pull/837/commits/6c2545bd9bf949c6db30ea9355816b37cd355b8a